File: Javascript/04-time-conversion.js
function timeConversion(minutes) {
	var hours = 0;

	while (minutes >= 60) {
		minutes -= 60;
		hours++;
	}

	if (minutes < 10) {
		minutes = "0" + minutes;
	}

	return hours + ":" + minutes;
}

console.log(timeConversion(360));

File: Python/04-time-conversion.py
def time_conversion(mins):
    hours = 0

    while mins >= 60:
        hours += 1
        mins -= 60

    if mins < 10:
        mins = "0" + str(mins)

    return "{hours}:{mins}".format(hours=hours, mins=mins)

print(time_conversion(365))